src/scss/text.scss

```diff
@@ -0,0 +1,6 @@
+hr {
```

Comment:
This is the horizontal divider, right? Not really to do with "text", can it just go in the left-nav stylings?

Reply:
[[refactor] Delete oddly named text.scss file](https://github.com/bitwarden/desktop/pull/1487/commits/e9c30d4232b7a4539ac8c3fde7bbdd3c24d511d3)